### ISSUE TITLE
[Feature Request] Saving Show Online & Offline

### ISSUE THREAD

**Issue by totalschaden** [SnosMe/awakened-poe-trade#953]
For Ruthless you always want to check Offline Seller Prices as the Playerbase is low, it sucks to have to switch to Online & Offline for every Check that you want to do.

Please add an Option to Save your Preference.

**Comment by nilskus** [comment]
agreed, even in crucible it's helpful to look at the offline offers if an item isn't selling.

**Comment by SethYokami** [comment]
We need that, seriously!

The problem is, the "one day ago" takes into consideration "WHEN the item was posted", instead of "WHEN the user last logged in", which is imo way more important, to know the user may or may not log in again to sell.
